detect_secrets/main.py

```diff
@@ -45,7 +45,7 @@ def main(argv=None):
             if args.import_filename:
                 _write_to_file(args.import_filename[0], output)
             else:
-                print(output)
+                print(output, end='')
```

Comment:
Let's make the change in `format_baseline_for_output` instead, and write the new line in `_write_to_file`.

That seems less of an anti-pattern.

Reply:
I'm on the fence about adding `+ '\n'` in 3 places, but I don't have strong feelings about it.

```diff 
 def _save_baseline_to_file(filename, data):  # pragma: no cover
     with open(filename, 'w') as f:
-        f.write(format_baseline_for_output(data))
+        f.write(format_baseline_for_output(data) + '\n')
...
-                _write_to_file(args.import_filename[0], output)
+                _write_to_file(
+                    args.import_filename[0],
+                    output + '\n'
+                )
...
     with open(filename, 'w') as f:
-        f.write(format_baseline_for_output(payload))
+        f.write(format_baseline_for_output(payload) + '\n')
```

Reply:
Maybe something along the lines of:

```
$ cat detect_secrets/core/common.py
def write_to_file(filename, data):
    with open(filename, 'w') as f:
        f.write(format_baseline_for_output(data) + '\n')
```

Reply:
✅